### ISSUE TITLE
[bug] Can't update languages...

### ISSUE THREAD

**Issue by vovpff** [YetiForceCompany/YetiForceCRM#11563]
# 🐞 bug report

### ⁉️ Describe the bug

![image](https://user-images.githubusercontent.com/8545791/65370370-a41f5a00-dc60-11e9-9f6a-89a04a1b33de.png)

:(

### 🌍  Your Environment 

| Environment		|	Version / Name	|
| ----------------	|	--------------	|
| YetiForce			|  5.2 sp6

**Comment by mariuszkrzaczkowski** [comment]
show logs

**Comment by vovpff** [comment]
@mariuszkrzaczkowski 
```
2019-09-21 16:43:35.412 [info][yii\db\Command::execute] - INSERT INTO `l_yf_settings_tracker_basic` (`user_id`, `type`, `module_name`, `record_id`, `date`, `action`) VALUES (1, 2, 'YetiForce', 0, '2019-09-21 16:43:35', 'Action:DownloadLanguage')
2019-09-21 16:43:36.7428 [warning][App\Fields\File] - Checked URL is not allowed: https://github.com/YetiForceCompany/YetiForceCRMLanguages/raw/master/5.2.15/ru-RU.zip | Client error: `GET https://github.com/YetiForceCompany/YetiForceCRMLanguages/raw/master/5.2.15/ru-RU.zip` resulted in a `404 Not Found` response:
```
Solved by replacing version from 5.2.15 to 5.2.0 in `/config/version.php/`
